python/ray/air/integrations/wandb.py

```diff
@@ -420,6 +420,12 @@ def run(self):
                 # Ignore HTTPError. Missing a few data points is not a
                 # big issue, as long as things eventually recover.
                 logger.warning("Failed to log result to w&b: {}".format(str(e)))
+            except FileNotFoundError as e:
+                logger.error(
+                    "FileNotFoundError: This result will not be logged to WandB. "
+                    "Possible reason: relative instead of absolute path for file %s",
+                    e,
+                )
```

Comment:
```suggestion
            except FileNotFoundError as e:
                logger.error(
                    f"FileNotFoundError: Failed to log result to Weights & Biases. "
                    f"Possible cause: relative file path used instead of an absolute path. "
                    f"Error: {e}"
                )
```